loopd/swapclient_server_test.go

```diff
@@ -143,3 +158,91 @@ func TestValidateLoopInRequest(t *testing.T) {
 		})
 	}
 }
+
+// TestValidateLoopOutRequest tests validation of loop out requests.
+func TestValidateLoopOutRequest(t *testing.T) {
+	tests := []struct {
+		name           string
+		chain          chaincfg.Params
+		confTarget     int32
+		destAddr       btcutil.Address
+		label          string
+		err            error
+		expectedTarget int32
+	}{
+		{
+			name:           "mainnet address with mainnet backend",
+			chain:          chaincfg.MainNetParams,
+			destAddr:       mainnetAddr,
+			label:          "label ok",
+			confTarget:     2,
+			err:            nil,
+			expectedTarget: 2,
+		},
+		{
+			name:           "mainnet address with testnet backend",
+			chain:          chaincfg.TestNet3Params,
+			destAddr:       mainnetAddr,
+			label:          "label ok",
+			confTarget:     2,
+			err:            errIncorrectChain,
+			expectedTarget: 0,
+		},
+		{
+			name:           "testnet address with testnet backend",
+			chain:          chaincfg.TestNet3Params,
+			destAddr:       testnetAddr,
+			label:          "label ok",
+			confTarget:     2,
+			err:            nil,
+			expectedTarget: 2,
+		},
+		{
+			name:           "testnet address with mainnet backend",
+			chain:          chaincfg.MainNetParams,
+			destAddr:       testnetAddr,
+			label:          "label ok",
+			confTarget:     2,
+			err:            errIncorrectChain,
+			expectedTarget: 0,
+		},
+		{
+			name:           "invalid label",
+			chain:          chaincfg.MainNetParams,
+			destAddr:       mainnetAddr,
+			label:          labels.Reserved,
+			confTarget:     2,
+			err:            labels.ErrReservedPrefix,
+			expectedTarget: 0,
+		},
+		{
+			name:           "invalid conf target",
+			chain:          chaincfg.MainNetParams,
+			destAddr:       mainnetAddr,
+			label:          "label ok",
+			confTarget:     1,
+			err:            errConfTargetTooLow,
+			expectedTarget: 0,
+		},
+		{
+			name:           "default conf target",
+			chain:          chaincfg.MainNetParams,
+			destAddr:       mainnetAddr,
+			label:          "label ok",
+			confTarget:     0,
+			err:            nil,
+			expectedTarget: 9,
+		},
+	}
+
+	for _, test := range tests {
+		test := test
```

Comment:
Only need this for `t.Parallel()` so that the `test` var doesn't get overwritten in parallel test runs. Might as well make these parallel.